mqtt/mqtt-broker/src/lib.rs

```diff
@@ -161,6 +161,13 @@ impl ConnReq {
         (self.peer_addr, self.connect, self.handle)
     }
 }
+#[derive(Debug, Serialize, Deserialize)]
+pub struct ServiceIdentity {
```

Comment:
as discussed offline, we would need to come up with more generic approach for Authorization updates. One option that I played with is in this commit (based from your branch) - https://github.com/vadim-kovalyov/iotedge/commit/15ed66fa223a7c6e9cd1e8c74009c8b2563f1f76

Feel free to just take it and patch your branch.